thinkcompany/vue-a11y-announcer
# Feature/update primary package versions

## What is this PR for?

This updates the version of Vue to the latest version

## How to test

1. Pull this branch down
1. Run `npm install`
1. Run `npm start`
1. Using a screen reader such as NVDA or VoiceOver, navigate to the button in the document and interact with it.
1. You should here `Here's a new announcement!` announced by the screen reader

## Additional Notes:
VueLoaderPlugin is now required so that webpack can compile vue template files.
